README.md

```diff
@@ -184,9 +184,22 @@ const transactionStatus = xrpClient.getPaymentStatus(transactionHash); // Transa
 **Note:** The example transactionHash may lead to a "Transaction not found." error because the TestNet is regularly reset, or the accessed node may only maintain one
 month of history.  Recent transaction hashes can be found in the XRP Ledger Explorer: https://livenet.xrpl.org/
 
+#### Payment history
+
+An `XRPClient` can return a list of payments to and from an account.
+
+```javascript
+const { XRPClient } = require("xpring-js");
+
+const remoteURL = "alpha.test.xrp.xpring.io:50051"; // TestNet URL, use alpha.xrp.xpring.io:50051 for MainNet
```

Comment:
nit: I think we capitalize these as `Testnet` and `Mainnet`

Reply:
standardized throughout both READMEs!